src/variables/apis/index.ts

```diff
@@ -0,0 +1,24 @@
+// Utils
+import {downloadTextFile} from 'src/shared/utils/download'
+
+import {postTemplatesExport} from 'src/client'
+
+export const downloadVariableTemplate = async (variable): Promise<void> => {
+  const resp = await postTemplatesExport({
+    data: {
+      resources: [
+        {
+          kind: 'Variable',
+          id: variable.id,
+        },
+      ],
+    },
+  })
+
+  if (resp.status === 500) {
```

Comment:
it'll be more future proof to say something like

```ts
if (resp.status >= 300) {
}
```

If in the future, a 400 level error is added, it'll be caught by `>= 300`, but not by `=== 500`

Reply:
Looks like our types define that won't be throwing any errors other than `200`, `201`, `500`.

As seen in the generatedRoutes definition:

```
type PostTemplatesApplyResult =
  | PostTemplatesApplyOKResult
  | PostTemplatesApplyCreatedResult
  | PostTemplatesApplyDefaultResult

interface PostTemplatesApplyOKResult {
  status: 200
  headers: Headers
  data: TemplateSummary
}

interface PostTemplatesApplyCreatedResult {
  status: 201
  headers: Headers
  data: TemplateSummary
}

interface PostTemplatesApplyDefaultResult {
  status: 500
  headers: Headers
  data: Error
}
```